### PR TITLE
Chat page shortcuts

### LINKED ISSUE #448: Add hotkeys at main window
## Parent Task: 
#446 - Add hotkeys 

## Description:
Add next hotkeys:

- Alt+n - switch channel to next
- Alt+p - switch channel to previous
- Shift+C - open channels list 
- Shift+Enter - send message

## Acceptance Criteria:
Hotkeys are working

### PR DESCRIPTION
**Pull request by Sirzhatina**

### CODE REVIEW

File: Client/Client.Qt/Widgets/InputFields.cpp [hunk 1]
```diff
@@ -139,3 +139,16 @@ void FlatTextEdit::paintEvent(QPaintEvent* event)
...
+{
+    if ((keyEvent->modifiers() & Qt::SHIFT) && keyEvent->key() == Qt::Key_Return)
+    {
+        emit messageSent();
+    }
+    else
+    {
+        QTextEdit::keyPressEvent(keyEvent);
+    }
+}
```
Comment (GuskoMaksim) [inline Client/Client.Qt/Widgets/InputFields.cpp:154]: Add a line break at the end, the rule described on WIKI

File: Client/Client.Qt/Widgets/TextEdit.cpp [hunk 2]
```diff
@@ -34,6 +34,7 @@ TextEdit::TextEdit(QWidget* parent) : QWidget(parent), _settings(Settings::getIn
     _underlineButton->setClickCallback([&]() { styleButtonClick(_underlineSymbolOpen, _underlineSymbolClose); });
     _sendButton->setClickCallback([&]() { sendButtonClick(); });
     connect(_messageInput.get(), &FlatTextEdit::textChanged, this, &TextEdit::textChanged);
+    connect(_messageInput.get(), SIGNAL(messageSent()), this, SLOT(sendButtonClick()));
```
Comment (GuskoMaksim) [inline Client/Client.Qt/Widgets/TextEdit.cpp:37]: I think you should stick with the style of "connect" one line above

**Review by GuskoMaksim — CHANGES_REQUESTED** [review]

File: Client/Client.Qt/Widgets/InputFields.hpp [hunk 3]
```diff
@@ -3,6 +3,7 @@
 #include <QLineEdit>
 #include <QTextEdit>
 #include <Style/Styles.hpp>
+#include <QKeyEvent>
```
Comment (StasDash) [inline Client/Client.Qt/Widgets/InputFields.hpp:6]: This line is unnecessary(all events are included in source file)

File: Client/Client.Qt/Widgets/InputFields.hpp [hunk 4]
```diff
@@ -48,7 +49,11 @@ class FlatTextEdit : public QTextEdit
     void paintEvent(QPaintEvent* paintEvent) override;
     /// Handle resize
     void resizeEvent(QResizeEvent* resizeEvent) override;
+    /// Handles pressing Shift+Enter combination
+    void keyPressEvent(QKeyEvent* keyEvent) override;
 
+signals:
+    void messageSent();
```
Comment (StasDash) [inline Client/Client.Qt/Widgets/InputFields.hpp:56]: This a common-case widget, so better common-name required

**Review by StasDash — CHANGES_REQUESTED** [review]

**Review by GuskoMaksim — APPROVED** [review]

**Review by GuskoMaksim — APPROVED** [review]

**Review by StasDash — APPROVED** [review]
Signals are for later

File: Client/Client.Qt/Widgets/InputFields.cpp [hunk 5]
```diff
@@ -139,3 +139,16 @@ void FlatTextEdit::paintEvent(QPaintEvent* event)
...
+
+void FlatTextEdit::keyPressEvent(QKeyEvent* keyEvent)
+{
+    if ((keyEvent->modifiers() & Qt::SHIFT) && keyEvent->key() == Qt::Key_Return)
+    {
+        emit shiftAndEnterPressed();
+    }
+    else
+    {
+        QTextEdit::keyPressEvent(keyEvent);
```
Comment (Orange-hanter) [inline Client/Client.Qt/Widgets/InputFields.cpp:152]: Call parent handler in any case.

**Review by Orange-hanter — CHANGES_REQUESTED** [review]

File: Client/Client.Qt/Widgets/InputFields.cpp (hunk 5 above)
Comment (Sirzhatina) [reply Client/Client.Qt/Widgets/InputFields.cpp:152]: You mean removing else part, don't you?

**Review by Orange-hanter — APPROVED** [review]

**Pull request merged**